### ISSUE TITLE
dict.all in data_bin/how2asr

### ISSUE THREAD

**Issue by PipiZong** [forkarinda/MFN#3]
Hi,

Is dict.all generated from the whole training set of the How2 dataset? But when building a binary dataset for both validation and test set using preprocess.py, there is no token replaced by 'unk'. So actually, dict.all is generated based on all dataset including train/val/test sets?
Thanks!

**Comment by balabanahei** [comment]
Hi, 

Yes, dict.all is generated based on all dataset including train/val/test sets.
